decaf377-ka/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "decaf377-ka"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ark-ff = { git = "https://github.com/penumbra-zone/algebra", branch = "ours" }
+decaf377 = { git = "https://github.com/penumbra-zone/decaf377" }
+rand_core = { version = "0.6.3", features = ["getrandom"] }
```

Comment:
what's the `getrandom` feature for? it doesn't seem to be used in the code, am i missing something?

Reply:
ah, it's used in the tests (via `OsRng`) - I can move this into a dev dependencies block though

Reply:
(expansion of what I mean by "not worth it": it causes problems sufficient that there are entire tools whose purpose is to re-unify feature selections, like @sunshowers ' `cargo-hakari`: https://docs.rs/cargo-hakari/0.9.5/cargo_hakari/about/index.html#what-are-workspace-hack-crates )